protocol/chainlib/chainlib.go

```diff
@@ -60,6 +60,7 @@ type ChainMessage interface {
 	GetServiceApi() *spectypes.ServiceApi
 	GetInterface() *spectypes.ApiInterface
 	RequestedBlock() int64
+	GetAverageBlockTime() int64
```

Comment:
this is not message specific therefore doesn;t belong here, also you have it in chainBlockStats